.github/renovate.json

```diff
@@ -8,9 +8,6 @@
   "schedule": [
     "monthly"
   ],
```

Comment:
Is there a way to configure it to only look at production dependencies? And what severity level does it care about?

Our current npm ci reports 19 vulnerabilities: https://github.com/ni/nimble/actions/runs/6537395490/job/17751139631?pr=1605#step:7:67

Is merging this going to create 19 PRs?

Reply:
I see the default strategy will update the lock file only so it should be able to merge immediately if it can: https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts

aight lets try it out

Reply:
Oh wait, should we do in-range-only? https://docs.renovatebot.com/configuration-options/#rangestrategy

Sounds like the default of update-lockfile might create package.json updates if needed which then need change files. Guess we can try either.

Reply:
I think I'll keep it at update-lockfile since that's what we use for our existing Renovate configuration for non-security updates. I'm hoping it'll bias towards only touching the lockfile but will still create package.json updates if needed. But if it gets too aggressive about touching the package file then I agree in-range-only would be a good idea.